Docs/source/pages/upgrade-guides/v-1-8-0.rst

```diff
@@ -72,7 +72,7 @@ Replace:
     <PropertyGroup>
         <PackageId>AltUnityDriver</PackageId>
         <TargetFrameworks>netstandard2.0;net5.0</TargetFrameworks>
-        <RootNamespace>Altom.AltUnityDriver</RootNamespace>
+        <RootNamespace>AltTester.AltUnityDriver</RootNamespace>
```

Comment:
see my previous comment

Reply:
the change needs to be made in the `With` section, not the `Replace` one, as I highlighted with a Red pen on my first comment thread with screenshots